src/use-cases/hybrid-search.md

```diff
@@ -0,0 +1,122 @@
+# Hybrid Search
+
+We will show you how to use `pgvecto.rs` to implement hybrid search in this section.
```

Comment:
We can advice users to set search mode to `vbase`.

Reply:
Is it necessary to mention it in the `hybrid search` section? It may add extra complexity to the doc.

Reply:
I'm not sure.

Reply:
cc @VoVAllen

Reply:
I don't think vbase is necessary in the context of hybrid search. Because usually people just union the results for further reranking steps. 

If the query looks like `SELECT xxx from xxx order by xxx WHERE docs include 'cat'`, then vbase is recommended.